examples/jsm/renderers/common/CanvasRenderTarget.js

```diff
@@ -0,0 +1,187 @@
+import { DepthTexture, Vector4, REVISION, createCanvasElement } from 'three';
+
+class CanvasRenderTarget {
```

Comment:
Shouldn't it extend `RenderTarget`?

Reply:
I suspect that would confuse things, this is intended to look more like the Renderer API, and RenderTarget creates an unneeded Texture() etc and has stencilBuffer etc.  Open to suggestions though.

It does need to extend EventDispatcher though of course, which I have added but not yet pushed